fedora/python-ogr.spec

```diff
@@ -0,0 +1,51 @@
+%global srcname ogr
+
+Name:           python-%{srcname}
+Version:        0.11.1
+Release:        %{?autorel}%{!?autorel:1}
+Summary:        One API for multiple git forges
+
+License:        MIT
+URL:            https://github.com/packit-service/ogr
+Source0:        %{pypi_source}
+BuildArch:      noarch
+
+BuildRequires:  python3-devel
+BuildRequires:  python3dist(setuptools)
+BuildRequires:  python3dist(setuptools-scm)
+BuildRequires:  python3dist(setuptools-scm-git-archive)
+
+%description
+One Git library to Rule!
+
+%package -n     python3-%{srcname}
+Summary:        %{summary}
+%{?python_provide:%python_provide python3-%{srcname}}
+
+%description -n python3-%{srcname}
+One Git library to Rule!
+
+
+%prep
+%autosetup -n ogr-0.11.1
```

Comment:
Sorry for being late for the party, but why can't we use `%{srcname}-%{version}`?
I don't see it mentioned in https://docs.pagure.org/Fedora-Infra.rpmautospec/principle.html

Reply:
because I'm an ignorant moron, obviously https://github.com/packit-service/ogr/pull/388